pkg/maps/metricsmap/metricsmap.go

```diff
@@ -59,6 +81,31 @@ func (v *Value) String() string {
 	return fmt.Sprintf("count:%d bytes:%d", v.Count, v.Bytes)
 }
 
+// RequestCount returns the drop/forward count in a human readable string format
+func (v *Value) RequestCount() string {
+	return strconv.FormatUint(v.Count, 10)
+}
+
+// RequestBytes returns drop/forward bytes in a human readable string format
+func (v *Value) RequestBytes() string {
+	return strconv.FormatUint(v.Bytes, 10)
+}
+
+// IsDrop checks if the reason is drop or not.
+func (k *Key) IsDrop() bool {
+	return k.Reason != 0
+}
+
+// CountFloat converts the request count to float
+func (v *Value) CountFloat() float64 {
+	return float64(v.Count)
+}
+
+// BytesFloat converts the request bytes to float
+func (v *Value) BytesFloat() float64 {
```

Comment:
Where is this function used?

Reply:
We had initially used this to send Bytes metrics too. I can remove this

Reply:
done